homeassistant/components/evohome/__init__.py

```diff
@@ -523,6 +522,11 @@ def get_session_id(client_v1: ev1.EvohomeClient) -> str | None:
                 ),
                 err,
             )
+            self.temps = {}  # high-precision temps now stale
+
+        except Exception:  # pylint: disable=broad-except
```

Comment:
I don't believe we need to disable pylint if we raise the exception again in the except block.